COLLABORATOR_GUIDE.md

```diff
@@ -548,7 +548,7 @@ $ npm install -g node-core-utils
 $ git node land $PRID
 ```
 
-If it's the first time you have used `node-core-utils`, you will be prompted
+If it's the first-time you have used `node-core-utils`, you will be prompted
```

Comment:
@jagannathBhat I think this one was correct, there is no word for which this is an attribute? :)

Reply:
Fun fact: new GitHub commit proposal feature does not work for such un-change cases.

Reply:
I agree, this change should be reverted.

Reply:
```suggestion
If it's the first time you have used `node-core-utils`, you will be prompted
```

I think it works.

Reply:
```suggestion
If it's the first time you have used `node-core-utils`, you will be prompted
```

Reply:
Sorry. It just was not shown in the preview properly, so I was confused.